power_grid_model_c/power_grid_model_c/power_grid_model_c.cpp

```diff
@@ -385,6 +394,8 @@ void PGM_calculate(PGM_Handle* handle, PGM_PowerGridModel* model, PGM_Options co
                         update_dataset, opt->threading);
                 }
                 break;
+            case PGM_short_circuit:
+                [[fallthrough]];
```

Comment:
If you fall through, it will still throw error? Maybe it should be `break` here. And add a TODO if we implement the actual C++ method.

Reply:
`[[fallthrough]]` indeed implies executing the next statement (by design of the attribute)

since the next statement already is a `not implemented` anyway, i do think adding it like this is better. if you think it's a good idea to also already _create_ (not implement) the C++ method, then indeed we can remove this `[[fallthrough]]` statement, but otherwise, I would keep it for the time being.

Reply:
@TonyXiang8787 if you agree with this change, this PR is ready for merge, regardless of the implementation of the validation (discussion tomorrow) itself

Reply:
agreed, I will go through this PR